src/nvim/ex_cmds.c

```diff
@@ -5846,3 +5914,286 @@ void set_context_in_sign_cmd(expand_T *xp, char_u *arg)
     }
   }
 }
+
+/// function called after a live command to get back to
+/// a normal state
+/// validate can take two values :
+///     0 if the live_command has not been validated
+///     1 if the user has validated the command
```

Comment:
Missing `@param` descriptions. Sentenses should start with capital letter. `{` below should be on its own line.